src/test/java/com/google/cloud/tools/ide/login/AccountRosterTest.java

```diff
@@ -0,0 +1,114 @@
+package com.google.cloud.tools.ide.login;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+import com.google.api.client.auth.oauth2.Credential;
+import org.junit.Test;
+
+public class AccountRosterTest {
+
```

Comment:
As we discussed offline, each test gets a fresh `AccountRoster` instance, so we are fine.